RELEASES.md

```diff
@@ -1,3 +1,122 @@
+Version 1.43.0 (2020-04-23)
+==========================
+
+Language
+--------
+- [Fixed using binary operations with `&{number}` (e.g. `&1.0`) not having
+  the correctly inferred type.][68129]
+
+**Syntax only changes**
+- [Allow `type Foo: Ord` syntactically.][69361]
+- [Unify item parsing & filter illegal item kinds.][69366]
+- [Fuse associated and extern items up to defaultness.][69194]
+- [Permit attributes on `if` expressions.][69201]
+- [Syntactically allow `self` in all `fn` contexts.][68764]
+- [Merge `fn` syntax + cleanup item parsing.][68728]
+
+These are still rejected *semantically*, so you will likely receive an error but
+these changes can be seen and parsed by procedural macros and
+conditional compilation.
+
+
+Compiler
+--------
+- [You can now pass multiple lint flags to rustc to override the previous
+  flags.][67885] For example; `rustc -D unused -A unused-variables` denies
+  everything in the `unused` lint group unused except `unused-variables` which
+  is explicitly allowed. Passing `rustc -A unused-variables -D unused` denies
+  everything in the `unused` lint group **including** `unused-variables` since
+  the allow flag is specified before the deny flag (and therefore overridden).
+- [rustc will now prefer your system MinGW libraries over its bundled libraries
+  if they are available on `windows-gnu`.][67429]
+- [rustc now buffers errors/warnings printed in JSON.][69227]
+
+Libraries
+---------
+- [`Arc<[T; N]>`, `Box<[T; N]>`, and `Rc<[T; N]>`, now implement
+  `TryFrom<Arc<[T]>>`,`TryFrom<Box<[T]>>`, and `TryFrom<Rc<[T]>>`
+  respectively.][69538] Where `N` is `0..=32`.
```

Comment:
I think that this is going to be unclear when rendered, because of the first `.` in the link.
I'd suggest:
```suggestion
  respectively.][69538] These conversions succeed when `N` is `0..=32`.
```